BrowserKit/Sources/ComponentLibrary/Buttons/LinkButton.swift

```diff
@@ -8,27 +8,23 @@ import UIKit
 public class LinkButton: ResizableButton, ThemeApplicable {
     private struct UX {
         static let buttonCornerRadius: CGFloat = 13
-        static let buttonVerticalInset: CGFloat = 12
-        static let buttonHorizontalInset: CGFloat = 16
     }
 
     override init(frame: CGRect) {
         super.init(frame: frame)
 
         backgroundColor = .clear
         titleLabel?.adjustsFontForContentSizeCategory = true
-        contentEdgeInsets = UIEdgeInsets(top: UX.buttonVerticalInset,
-                                         left: UX.buttonHorizontalInset,
-                                         bottom: UX.buttonVerticalInset,
-                                         right: UX.buttonHorizontalInset)
     }
 
     public func configure(viewModel: LinkButtonViewModel) {
         accessibilityIdentifier = viewModel.a11yIdentifier
         setTitle(viewModel.title, for: .normal)
         titleLabel?.font = DefaultDynamicFontHelper.preferredFont(withTextStyle: .body,
                                                                   size: viewModel.fontSize)
+        contentEdgeInsets = viewModel.contentEdgeInsets
         titleLabel?.textAlignment = viewModel.textAlignment
+        contentHorizontalAlignment = viewModel.contentHorizontalAlignment
```

Comment:
Do we need `titleLabel?.textAlignment` if we have `contentHorizontalAlignment`?

Reply:
I think it can be removed, shall I go ahead and make this change?

Reply:
If we get same result, then I would say yes

Reply:
yes, the results are the same and so updated the PR.